plugins/callback/foreman.py

```diff
@@ -31,6 +40,16 @@
         ini:
           - section: callback_foreman
             key: url
+      proxy_url:
+        description:
+          - URL of the Foreman Smart Proxy server.
+        env:
+          - name: FOREMAN_PROXY_URL
+        required: True
+        default: https://localhost:8443
```

Comment:
Shall I delete the default value also from foreman_url?

Hmm I am wondering - should it be required since there are two options now. Previously it made sense, there was just a one URL, now there are two. Although the plan is to move facts too into a plugin. 

I think let's just make both required, we should be working on facts in this release.

Reply:
please don't touch `foreman_url`, it's still there for not breaking backwards compatibility (i.e.: not changing defaults).

but you raise a good question, it should probaly not be `required: True`

Reply:
Can do a different PR if you want me to remove the required flag. But this should be deprecated pretty soon - I plan working also on the facts endpoint so everything should go through the proxy in the future.